api/src/main/java/org/openmrs/ui/framework/resource/ModuleResourceProvider.java

```diff
@@ -36,8 +41,13 @@
 	 */
 	@Override
 	public File getResource(String path) {
-		if (resourceShortcuts != null && resourceShortcuts.containsKey(path))
+		if (resourceShortcuts != null && resourceShortcuts.containsKey(path)) {
 			path = resourceShortcuts.get(path);
+		}
+
+		if (path == null || new File(path).isAbsolute()) {
```

Comment:
`new File(path).isAbsolute()` what's this do? Is this a Java idiom I don't know because I'm not really a Java guy, or does this warrant a comment?

Reply:
This is the Java idiom for testing if a path describes an absolute path for the default file system (at least on Java 6 and below). I don't think it's particularly remarkable.

Reply:
Right, that much is clear, but why would we return null in that case?

Reply:
Nice, that's helpful, thank you!